uo_sys_framework/Core/cfgFunctions/EventHandlers/fn_KilledHandler.sqf

```diff
@@ -33,46 +33,46 @@ switch (side player) do {
     [_unit] joinSilent grpNull;
 }, [_unit], 2] call CBA_fnc_WaitAndExecute;
 
-if (EGETMVAR(Spectator,Killcam_Enabled,true)) then {
```

Comment:
Yeah we can assume everyone is going to use killcam and instant death, have these settings been removed from the 3den menus?

Reply:
Spectator,Killcam_Enabled does not exist in the anymore. You mad it factionized to Killcam_Enabled_Blufor.

Reply:
Then those faction settings don't have an effect and don't need to be in those menus. Can you consider killcam ever being needing to be disabled?

Reply:
I personally would leave the option in. You never know what someone might want and it doesn't have any major impact.